### PR TITLE
rewording of i18n/l10n and linkage to other efforts

### PR DESCRIPTION
**Pull request by srl295**
some adjustments

I actually wondered about trying to bring in more of https://github.com/nodejs/node/wiki/Intl#q-how-do-the-pieces-fit-together--how-do-i-know-who-is-responsible-for-the-bug here, but that is probably a separate effort

### CODE REVIEW

**Comment by srl295** [comment]
In the Vai language in Liberia, February is spelled ꕒꕡꖝꖕ. This data is collected in CLDR's survey tool, where it makes its way into a CLDR release and finally into ICU, where Node picks it up and integrates it where v8 can call it. 

Thus:

```
> new Date().toLocaleDateString(["vai"], {month: "long"})
'ꕒꕡꖝꖕ'
> new Date().toLocaleDateString(["en"], {month: "long"})
'February'
```

Making sure that groups (such as the Vai language team) know how to get these translation fixed is where CLDR liaison comes in. See also #13

File: README.md [hunk 1]
```diff
@@ -2,42 +2,45 @@
...
+#### What i18n (Internationalization) means to us
+Maintaining the ability for Node.js to [effectively support](https://nodejs.org/api/intl.html#intl_internationalization_support) the cultural & socio-linguistic preferences of all international users, through:
+* [Unicode](https://unicode.org) processing and related services to support text written in all human languages.
+* APIs and implementations which support the specific cultural & socio-linguistic preferences, such as localized methods for displaying dates & times.
+* The ability for [Node.js](https://github.com/nodejs/node) and its related modules & applications to be translated into distinct human languages.
 
-#### What l10n means to us
-* The ability for [Node.js](https://github.com/nodejs/node) and its related modules & applications to be translated into distinct human languages. 
+#### What l10n (Localization) means to us
+* Making the Node.js project's API documentation, site, and tutorial content available in every language.
```
Comment (bnoordhuis) [inline README.md]: Ambitious goal. :-)

Comment (srl295) [reply README.md]: @bnoordhuis Better get started!

Comment (obensource) [reply README.md]: @bnoordhuis @srl295 yep! :)

File: README.md [hunk 2]
```diff
@@ -2,42 +2,45 @@
...
+In order to ensure best practices, this working group may work directly with representatives of similar and related i18n efforts from other external projects. For example:
 * [Electron i18n](https://github.com/electron/electron-i18n)
 * [W3C i18n Working Group](https://www.w3.org/International/core/Overview)
-* [V8 i18n](https://github.com/v8/v8/wiki/i18n-support)
+* [ECMA-402](https://github.com/tc39/ecma402) (Together with ECMA-262, specifies the i18n features of JavaScript)
+* [V8 i18n](https://github.com/v8/v8/wiki/i18n-support) (implements ECMA-402 in Node.js)
+* [Unicode](https://unicode.org):
+  - [The Unicode Standard](https://unicode.org/standard/) (defines how text is encoded)
+  - [CLDR](https://cldr.unicode.org) (Common Locale Data Repository)
+  - [ICU](https://icu-project.org) (C/C++ i18n library used by v8 and Node.js)
```
Comment (obensource) [inline README.md]: Thanks for adding CLDR/ICU to our relationships. Excited to support these efforts. 👍

**Comment by obensource** [comment]
@srl295 
* I think creating some easy to read documentation here concerning ['how-do-the-peices-fit-together'](https://github.com/nodejs/node/wiki/Intl#q-how-do-the-pieces-fit-together--how-do-i-know-who-is-responsible-for-the-bug) would actually be quite helpful for this group in order to gain a better understanding of the Intl process in general, and help us understand our context. Would be a great thing to add as an issue and maybe we can solidify what falls under our scope, or if you'd like to put in a PR we can discuss it more there too. 😎
* Great point about supporting the Vai language team and maintaining a relationship with the CLDR project. The more liaison relationships we create to support people the better. 👍

**Comment by obensource** [comment]
@srl295 feel free to merge at your leisure! :)

**Comment by srl295** [comment]
Ok! Vai was just an example.

**Comment by srl295** [comment]
@obensource I don't have write access here

**Comment by obensource** [comment]
@srl295 re Vai: gotcha lol–I'm just excited about our endeavor. Thanks for clarifying. :)

I'm not sure why you don't have write access, you should given that you're an org member & team member. Since @bnb has admin privileges, he'll check as soon as he can. 👍

For now I'm going to merge this, but hopefully you'll have all you need for that asap. 🍻

**Comment by obensource** [comment]
@srl295 turns out this repo was set to read-only access for the team. That's been changed to allow write access as well now. (thanks @bnb) 👍

**Pull request merged**